llvm/lib/Target/Xtensa/Xtensa.h

```diff
@@ -0,0 +1,30 @@
+//===- Xtensa.h - Top-level interface for Xtensa representation -*- C++ -*-===//
+//
+//                     The LLVM Compiler Infrastructure
```

Comment:
Current license header template doesn't include this string. I don't know if this is important or not.

Reply:
Thank you very much for comments. I removed these strings from all new files.